src/libs/actions/Network.ts

```diff
@@ -1,26 +1,18 @@
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 
-/**
- * @param {Boolean} isOffline
- */
-function setIsOffline(isOffline) {
+function setIsOffline(isOffline: boolean): void {
```

Comment:
Same here, we can remove `void` typing of these functions. WDYT?

Reply:
Done! @fabioh8010